src/dev/precommit_hook/casing_check_config.js

```diff
@@ -116,6 +116,7 @@ export const IGNORE_DIRECTORY_GLOBS = [
   'src/babel-*',
   'packages/*',
   'packages/core/*/*',
+  'x-pack/packages/ai-infra/*',
```

Comment:
@elastic/kibana-operations the ai-infra packages will live in that folder, and I kinda like my package folders to follow their names (e.g `ai-infra/kb-artifact-builder` instead of `ai-infra/kb_artifact_builder`), so I took the liberty to add that pattern. 

Please tell me if you'd prefer me to just go with underscores.

Reply:
No issues with it